sdk/react/panels/videoView.js

```diff
@@ -304,9 +304,8 @@ var VideoView = React.createClass({
        <ActivityIndicator
           style={loadingStyle}
           size="large"
-      />
-        );
-      }
+       />
+      );
```

Comment:
No I changed some code in last pull request and missed to remove this one. So checking it in now:)